outlierdetect/outlierdetect.py

```diff
@@ -238,21 +240,23 @@ def _normalize(self, value_dict):
 
 ########################################## Helper functions ########################################
 
-def _normalize_counts(counts, val=1):
+def _normalize_counts(counts, val=1, percent_format=False):
     """Normalizes a dictionary of counts, such as those returned by _get_frequencies().
 
     Args:
         counts: a dictionary mapping value -> count.
         val: the number the counts should add up to.
-    
+        percent_format: boolean that determines if the frequencies are converted to a cleaner, rounded percent format.
```

Comment:
This is just a display issue, correct? Assuming yes, I don't think it belongs in the `_normalize_counts` method. as you can just do the rounding elsewhere, right?

Reply:
> I suspect you were using the file inline (instead of importing it as a library) and calling `_normalize_counts()` directly. Is that correct?

Initially I just copied over the function and made the modifications myself when I was testing it out. When making the changes for this pull request, I decided to just edit the function itself in the library.

Though I see what you're saying, about just doing it outside of the algorithm. Was just trying to see how much I could add to the library to make it easier for formatting quickly. I have a separate method still - format_scores() - that still does some quick & easy formatting, and I can add a line or two to that to get the same results as this percent_format thing.

Thanks!